hapi-fees/src/test/java/com/hedera/services/usage/token/TokenGetAccountNftInfosUsageTest.java

```diff
@@ -0,0 +1,79 @@
+package com.hedera.services.usage.token;
+
+/*-
+ * ‌
+ * Hedera Services API Fees
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.google.protobuf.ByteString;
+import com.hederahashgraph.api.proto.java.AccountID;
+import com.hederahashgraph.api.proto.java.Query;
+import com.hederahashgraph.api.proto.java.TokenGetAccountNftInfosQuery;
+import com.hederahashgraph.fee.FeeBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static com.hedera.services.usage.token.entities.NftEntitySizes.NFT_ENTITY_SIZES;
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_ENTITY_ID_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_QUERY_RES_HEADER;
+import static com.hederahashgraph.fee.FeeBuilder.INT_SIZE;
+import static org.junit.Assert.assertEquals;
+
+public class TokenGetAccountNftInfosUsageTest {
+    TokenGetAccountNftInfosUsage subject;
+    AccountID id;
+    List<ByteString> metadata;
+
+    @BeforeEach
+    public void setup() {
+        metadata = List.of(ByteString.copyFromUtf8("some metadata"));
+        id = AccountID.newBuilder()
+                .setShardNum(0)
+                .setRealmNum(0)
+                .setAccountNum(1)
+                .build();
+        subject = TokenGetAccountNftInfosUsage.newEstimate(tokenQuery());
+    }
+
+    @Test
+    public void assessesEverything() {
+        // given:
+        subject.givenMetadata(metadata);
+
+        // when:
+        var usage = subject.get();
+        int additionalRb = metadata.stream().mapToInt(ByteString::size).sum();
```

Comment:
This should be counting metadata as a whole, not the individual bytes in it. It should be:

`int additionalRb = 0;
for (ByteString m : metadata) {
	additionalRb += metadata.size();
}`

Reply:
So you are saying if we have 2 elements in the list, this for loop should calculate a result equal to 4?

Reply:
The logic on ln. 62 gets every single metadata, counts its bytes and sums all of their sizes to give the estimated bytes.
e.g.
a = "abc"
b = "cbdeg"
result = a.size() + b.size() -> 8, not List.of(a, b).size() * 2